_posts/blog/2017-07-19-Actions-for-android-devs.md

```diff
@@ -8,19 +8,18 @@ tags: [Google Assistant, Actions on Google, conversational interfaces, Android,
 
 # Google Assistant Development for Android Devs (Part 1)
 
-![](https://developers.google.com/actions/images/surfaces-hero.svg)
+![](https://storage.googleapis.com/banjotms.appspot.com/assistant_surfaces.svg)
 
-There is a lot of buzz and excitment around the [Google Assistant Actions](https://developers.google.com/actions/), and with good reason:  this is an entirely new platform for building experiences for a number of devices.  Building a conversational Action with Google Assistant means it will be available on Android, iOS, Google Home, and others as they com available.
+There is a lot of buzz and excitment around the [Google Assistant Actions](https://developers.google.com/actions/), and with good reason:  this is an entirely new platform for building experiences for a number of devices.  Building a conversational Action with Google Assistant means it will be available on Android, iOS, Google Home, and others as they become available.
 
 
-For the uninitiated, Actions on Google are third party "Actions" that can be built using Google's APIs.  Actions extend what the Google Assistant can do by letting user talk to them.  They can be accessed with trigger words such as 'Ok Google, talk to Ticketmaster', or 'Ok Google, ask ticketmaster find Yankee's games this weekend' (Ticketmaster Action coming soon!)
+For the uninitiated, Actions on Google are third-party apps that can be built using Google's APIs.  Actions extend what the Google Assistant can do by letting user talk to them.  They can be accessed with trigger words such as 'Ok Google, talk to Ticketmaster', or 'Ok Google, ask ticketmaster find Yankee's games this weekend'. (Ticketmaster Action coming soon!)
```

Comment:
I think we want to emphasize the conversational piece of this tool. How about we make the query a more natural sentence by adding more articles ...  "ask Ticketmaster to find a Yankee's game this weekend"?